tests/bugs/replicate/bug-1586020-mark-dirty-for-entry-txn-on-quorum-failure.t

```diff
@@ -9,10 +9,9 @@ function create_files {
         local i=1
         while (true)
         do
-                dd if=/dev/zero of=$M0/file$i bs=1M count=10
-                blksize1=`stat -c %b $B0/${V0}0/file$i`
-                blksize2=`stat -c %b $B0/${V0}1/file$i`
-                if [[ $blkize1 -eq 20480 ]] || [[ $blksize2 -eq 20480 ]]; then
+                touch $M0/file$i
+                if [ -e $B0/${V0}0/file$i ] || [ -e $B0/${V0}1/file$i ]; then
+                        dd if=/dev/zero of=$M0/file$i bs=1M count=10 2>/dev/null
```

Comment:
It will create too many files on the backend, sometimes it is possible the test case can take time more than the default time (200s).

Reply:
I have checked more, there is some issue in softserve VM so I was getting the wrong behavior. Please ignore my previous comment.